pkg/newlog/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG GOVER=1.12.4
+FROM golang:${GOVER}-alpine as build
+RUN apk add --no-cache   \
+        git=2.20.4-r0    \
+        gcc=8.3.0-r0     \
+        linux-headers=4.18.13-r1
+
+ENV GOFLAGS=-mod=vendor
+ENV GO111MODULE=on
+ENV CGO_ENABLED=1
+
+COPY ./  /newlog/.
+WORKDIR /newlog
+
+RUN go build -mod=vendor -o newlogd ./cmd
```

Comment:
what's the additional size impact of this container on rootfs and what kind of memory footprint are you seeing @naiming-zededa ?

Reply:
looked at the size of xen/kvm squash images, about 260Mbytes, on my device running debug, i got top showing newlogd with 111m in VSS,

Reply:
What is the RSS for rsyslogd in comparison? My understanding is that rsyslogd isn't started unless forced with the /config file. but running both of them might push small systems over the edge.

Reply:
only the docker part is running, not the daemon itself for rsyslog. below is example on ps grep for rsyslog
1262 root      0:00 /usr/bin/containerd-shim-runc-v2 -namespace services.linuxkit -id rsyslogd -address /run/containerd/containerd.sock
and the RSS is 1060:
ps -o pid,vsz,rss | grep 1262
 1262 110m 1060

Reply:
111m of heap is really not ideal since we're trying to stay under 500Mb @naiming-zededa -- not sure what to do about this just yet -- but it seems like a big deal to me and would definitely affect our ability to deliver on Dell 300x systems.

Reply:
@rvs @eriknordmark the 111m is the virtual memory the 'go' program allocate, even I made this newlogd to be only a simple for loop, it would have 100m VSZ reported. I also found if change to go version 1.15, the VSZ reported would shoot up to 700m! but the RSS, the usage of the memory is about 10m, and with the newlogd message is limited by the channel message, it will not grow much for real memory usage.